cognite/neat/explorer/explorer.py

```diff
@@ -357,41 +358,101 @@ def get_nodes_and_edges(request: NodesAndEdgesRequest):
     logging.info("Querying graph nodes and edges :")
     if "get_nodes_and_edges" in cache_store and request.cache:
         return cache_store["get_nodes_and_edges"]
-    nodes_filter = ""
-    edges_dst_filter = ""
-    edges_src_filter = ""
-
-    if len(request.node_class_filter) > 0:
-        nodes_filter = "VALUES ?node_class { " + " ".join([f"<{x}>" for x in request.node_class_filter]) + " }"
-
-    node_name_property = request.node_name_property or "cim:IdentifiedObject.name"
-    nodes_query = f"SELECT DISTINCT ?node_id ?node_class ?node_name WHERE \
-    {{ {nodes_filter} \
-    ?node_id {node_name_property} ?node_name . ?node_id rdf:type ?node_class }} "
-    if len(request.src_edge_filter) > 0:
-        edges_src_filter = "VALUES ?src_object_class { " + " ".join([f"<{x}>" for x in request.src_edge_filter]) + " }"
-    if len(request.dst_edge_filter) > 0:
-        edges_dst_filter = "VALUES ?dst_object_class { " + " ".join([f"<{x}>" for x in request.dst_edge_filter]) + " }"
-
-    edges_query = f"SELECT  ?src_object_ref ?conn ?dst_object_ref \
-        WHERE {{ \
-        {edges_src_filter} \
-        {edges_dst_filter} \
-        ?src_object_ref ?conn  ?dst_object_ref . \
-        ?src_object_ref  rdf:type ?src_object_class . \
-        ?dst_object_ref  rdf:type ?dst_object_class .\
-        }} "
-    logging.info(f"Nodes query : {nodes_query}")
-    logging.info(f"Edges query : {edges_query}")
-    nodes_result = get_data_from_graph(nodes_query, request.graph_name, request.workflow_name)
-    edges_result = get_data_from_graph(edges_query, request.graph_name, request.workflow_name)
+    nodes_result = {}
+    edges_result = {}
+    query = ""
+    elapsed_time_sec = 0
```

Comment:
These are not used (they are overwritten further down)
```suggestion
```

Reply:
That's inherited from other languages where variables must be initialised first